doc/interface/spatial_discretization_methods.rst

```diff
@@ -0,0 +1,127 @@
+.. spatial_discretization_methods:
+
+Spatial discretization methods
+==============================
+
+Group /input/model/unit_XXX/discretization/SPATIAL_METHOD - Details on the methods
+----------------------------------------------------------------------------------
+
+CADET offers two spatial discretization methods: Finite Volumes (FV) and Discontinuous Galerkin (DG).
+While both methods approximate the same solution to the same underlying model, they may differ in terms of computational performance.
+Generally, FV can be more performant for small problem sizes and solutions with steep gradients, while DG excels for large problem sizes and smooth solutions.
+
+In the following, we give a brief introduction to the numerical theory that is most relevant for the computational performance of the methods.
+Based on that theory and our experience, we give advice on which method to use in which scenario, how to identify the more performant method, and how to specify the discretization parameters.
+
+Discrete system size
+--------------------
+
+Numerical methods discretize the continuous (here: spatial) domain of the equations into a finite set of discrete points.
+Then, a system of equations is formulated for those points, and both system size and number of unknowns / degrees of freedom (DoF) are given by the number of discrete points.
+This system can be linear or non-linear, depending on the method (this will become important again in the section on smooth solutions).
+The wall clock time to compute the solution depends on the system size and, thus, on the number of discrete points.
+Conversely, the numerical solution is more accurate with more discrete points.
+Thus, we trade computation time for approximation accuracy by specifying the parameters that determine the number of discrete points.
+For the FV scheme, the number of axial discrete points in the column is given by the number of volume cells ``NCOL``.
+For the DG scheme, the number of axial discrete points in the column is given by the number of polynomial interpolation nodes (= ``POLYDEG`` + 1) times the number of DG elements ``NELEM``.
+At each axial discrete point, the LRMP and GRM additionally employ a particle.
+These particles are also discretized, which increases the total number of DOF per axial discrete point, especially for the GRM where particles are fully resolved.
```

Comment:
But particles are only discretized for the GRM, right?

Reply:
They are discretized by a single volume, i.e. no transport happens but its still turned into a discrete equation

Reply:
Ah got it. Thanks for the clarification!